app/services/create_subscriber_list_service.rb

```diff
@@ -43,14 +41,14 @@ def find_exact_query_params
     }
   end
 
-  def slugify(title)
-    slug = title.parameterize.truncate(255, omission: "", separator: "-")
+  def slug
+    result = params[:title].parameterize.truncate(255, omission: "", separator: "-")
```

Comment:
Do you think there's any risk of this being accidentally used multiple times now that it's named a bit more like a variable than a method? I wonder whether it's useful to memoize this to prevent `n` different slugs being created for the same thing in some horrible future?
Admittedly it's quite edgey as edge cases come.

Reply:
That's a fair point, and it makes sense to call out the expectation now. I've amended the last commit to memoise.